### ISSUE TITLE
allowed_methods doesn't work

### ISSUE THREAD

**Issue by samsonasik** [zendframework/zend-expressive#68]
I tried using : 
```php
    'routes' => [
        [
            'path' => '/',
            'middleware' => 'Application\HelloWorld',
            'allowed_methods' => [ 'GET' ],
        ],
   ],
```
When I tried to call "/" via POST method, it get blank page with 200 status.

**Comment by samsonasik** [comment]
or if I set 'allowed_methods' as ['POST'], if I call '/' with 'GET', I still get 200 status.

**Comment by ezimuel** [comment]
@samsonasik I think the PR https://github.com/zendframework/zend-expressive/pull/73 fixes this issue. Remember that you need to use also the PR https://github.com/zendframework/zend-psr7bridge/pull/2. Can you try, please? Thanks!

**Comment by samsonasik** [comment]
is zendframework/zend-psr7bridge required? we have in "suggest" in composer.json?


Warm regards,

Abdul Malik Ikhsan

Pada 18 Agt 2015, pukul 20.21, Enrico Zimuel <notifications@github.com> menulis:

**Comment by ezimuel** [comment]
@samsonasik it's used by Zf2 route adapter, and it's included in require-dev.

**Comment by samsonasik** [comment]
ok, I will try it now

**Comment by samsonasik** [comment]
@ezimuel I've just tried and get error: "500 Internal Server Error" : 
```
PHP Deprecated:  Automatically populating $HTTP_RAW_POST_DATA is deprecated and will be removed in a future version. To avoid this warning set 'always_populate_raw_post_data' to '-1' in php.ini and use the php://input stream instead. in Unknown on line 0
```

**Comment by samsonasik** [comment]
for note: my composer configuration pull your repo and branch: 
```
{
    "require": {
        "zendframework/zend-expressive": "dev-fix/40",
        "league/plates": "^3.1"
    },
    "repositories" : [ {
        "type" : "vcs",
        "url" : "git://github.com/ezimuel/zend-expressive.git"
        }
    ],
    "autoload":{
        "psr-4": {
            "Application\\": "src"
        }
    }
}
```

**Comment by samsonasik** [comment]
@ezimuel I've retried your repo and branch with disabled error handler template, and it works now.

**Comment by samsonasik** [comment]
it can be closed if #73 will be merged

**Comment by samsonasik** [comment]
oh, but still, in php log, I still get error: 
```
[Tue Aug 18 21:30:42 2015] PHP Deprecated:  Automatically populating $HTTP_RAW_POST_DATA is deprecated and will be removed in a future version. To avoid this warning set 'always_populate_raw_post_data' to '-1' in php.ini and use the php://input stream instead. in Unknown on line 0
```

**Comment by weierophinney** [comment]
@samsonasik That error is unrelated to zend-expressive or any ZF2 components, as we do not access that. Most likely you have an extension that's reading that value; are you trying the Z-Ray Standalone beta, by any chance?

**Comment by samsonasik** [comment]
ok, will try that if there is a chance

**Comment by samsonasik** [comment]
@weierophinney I've set in php.ini:
```
always_populate_raw_post_data=-1
```
and now the error is gone.

**Comment by samsonasik** [comment]
closed in favor of #73
